charts/kuberpult/values.yaml

```diff
@@ -278,6 +278,9 @@ auth:
     enabled: false
     # Indicates if dex is to be installed. If you want to use your own Dex instance do not enable this flag.
     installDex: false
+    #If kuberpult cannot find a role in the dex response, it will use the role "default".
+    #This is only recommended for when you want the simplest possible setup, or for testing purposes."
```

Comment:
```suggestion
    # If kuberpult cannot find a role in the dex response, it will use the role "default".
    # This is only recommended for when you want the simplest possible setup, or for testing purposes.
```